src/test/java/com/rocketden/main/mapper/ProblemMapperTests.java

```diff
@@ -40,7 +38,7 @@ public void entityToDto() {
 
         ProblemDto actual = ProblemMapper.toDto(expected);
 
-        assertEquals(expected.getId(), actual.getId());
+        assertEquals(expected.getProblemId(), actual.getProblemId());
```

Comment:
I will note that since you're using UUID generation, you don't know what form this ID takes. You corrected me in this PR that you should be able to use UUID with testing by just fetching the problemId from the object, but there still is this small note -- is this an issue, or no?

Reply:
I think it should hopefully be a non-issue, but yeah definitely something to consider before doing that other Trello task I think